src/mpid/ch3/include/mpidpost.h

```diff
@@ -208,4 +208,56 @@ int MPIDI_CH3I_Progress_deactivate_hook(int id);
 #define MPID_Progress_activate_hook(id_) MPIDI_CH3I_Progress_activate_hook(id_)
 #define MPID_Progress_deactivate_hook(id_) MPIDI_CH3I_Progress_deactivate_hook(id_)
 
+static inline int MPID_Test(MPIR_Request *request_ptr, int *flag,
```

Comment:
`MPL_STATIC_INLINE_PREIFX` instead of `static inline`?
Also, could you remind me why you chose to use functions instead of macros for these fallbacks?

Reply:
I think these options are equivalent in this case. And we plan to update these functions in near future anyway.

Reply:
I think these options are equivalent in this case. And we plan to update these functions in near future anyway.

Reply:
I updated using `MPL_STATIC_INLINE_PREFIX` for consistency